DevTools/HackStudio/Editor.cpp

```diff
@@ -94,6 +94,10 @@ void Editor::show_documentation_tooltip_if_available(const String& hovered_token
         return;
     }
 
+    if (m_documentation_tooltip_window->is_visible()) {
```

Comment:
If you move the mouse from one token to another, but both tokens have documentation pages, we should still load a new file.
We need to remember what documentation we've loaded and only skip loading if the same one is requested again :)